net/pfSense-pkg-siproxd/Makefile

```diff
@@ -2,7 +2,7 @@
 
 PORTNAME=	pfSense-pkg-siproxd
 PORTVERSION=	1.1.3
-PORTREVISION=	3
+PORTREVISION=	4
```

Comment:
This is a large enough change to warrant bumping `PORTVERSION` (which also means removing `PORTREVISION`)

Reply:
done